hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -509,6 +512,25 @@ public static void validateBucketName(String bucketName)
     }
   }
 
+  /**
+   * Verify bucket layout is a valid.
+   *
+   * @return The {@link BucketLayout} corresponding to the string.
+   * @throws ConfigurationException If the bucket layout is not valid.
+   */
+  public static BucketLayout validateBucketLayout(String bucketLayoutString) {
+    boolean bucketLayoutValid = Arrays.stream(BucketLayout.values())
+        .anyMatch(layout -> layout.name().equals(bucketLayoutString));
+    if (bucketLayoutValid) {
+      return BucketLayout.fromString(bucketLayoutString);
+    } else {
+      throw new ConfigurationException(bucketLayoutString +
+          " is not a valid default bucket layout. Supported values are " +
+          Arrays.stream(BucketLayout.values())
+              .map(Enum::toString).collect(Collectors.joining(", ")));
```

Comment:
Nit: validity check above uses `Enum#name()`, but exception message uses `toString()`.  These are equivalent by default, but `toString()` can be overridden (e.g. for i18n/l10n purposes).

```suggestion
              .map(Enum::name).collect(Collectors.joining(", ")));
```

Better yet, create a `SortedSet<String>` constant in `BucketLayout` with all names, and use it in both validity check and exception message.